kyuubi-hive-jdbc/pom.xml

```diff
@@ -171,6 +171,14 @@
     </dependencies>
 
     <build>
+
+        <resources>
+            <resource>
+                <filtering>true</filtering>
+                <directory>src/main/resources</directory>
+            </resource>
+        </resources>
```

Comment:
we need it? `src/main/resources` should be the default location for resources.

Reply:
without this:
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/6757692/219956247-e6726748-6c19-4b55-801b-c1b8e3128ff3.png">

Reply:
with this:

<img width="1091" alt="image" src="https://user-images.githubusercontent.com/6757692/219956360-b902f9c5-a942-4429-b576-54feb57cb214.png">

Reply:
Where and how (what goal or execution) this placeholder is filled?

Reply:
Not sure